src/fios.cpp

```diff
@@ -230,16 +230,6 @@ std::string FiosMakeHeightmapName(std::string_view name)
 	return FiosMakeFilename(_fios_path, name, fmt::format(".{}", GetCurrentScreenshotExtension()));
 }
 
-/**
- * Delete a file.
- * @param name Filename to delete.
- * @return Whether the file deletion was successful.
- */
-bool FiosDelete(std::string_view name)
-{
-	return FioRemove(FiosMakeSavegameName(name));
```

Comment:
looks like there's only a single other usage of FiosMakeSavegameName - is that still necessary?